.golangci.yml

```diff
@@ -64,6 +64,9 @@ issues:
     - path: scale_resolvers_test.go
       linters:
         - staticcheck
+    - path: kubernetes_workload_scaler_test.go
```

Comment:
What is the reason for this?

Reply:
I need to use `"sigs.k8s.io/controller-runtime/pkg/client/fake"` for the unit tests, and it's deprecated. I found that the solution applied for that in `scale_resolvers_test` was to disable the linter there, and I replicated it

Reply:
Have you tried the `NewClientBuilder` approach as it is suggested by the Deprecation message?

Reply:
Honestly, I don't have so much experience using Golang. Anyway, the suggestion that I get is to replace this package in favor of `pkg/envtest`
> package sigs.k8s.io/controller-runtime/pkg/client/fake is deprecated: please use pkg/envtest for testing. This package will be dropped before the v1.0.0 release. (SA1019)go-staticcheck

I have reviewed how it works and I feel that pkg/envtest is more for integration test than unit test, but I'm open to change it. I have seen also that there are both cases inside the code base. Honestly, I took this approach only because it was easier, sorry but idk if it is the best one :(

Reply:
okay, could you please at least add a short message with the reason to the .golangci.yml ? Similar way it has been done for other exlusions.

Reply:
sure, I didn't add it because the reason is exactly the same as the previous one :sweat_smile: but you're righ, it's better if it's explicitly set